src/vs/workbench/contrib/chat/browser/agentSessions/agentSessionsModel.ts

```diff
@@ -404,14 +400,18 @@ class AgentSessionsCache {
 				archived: session.archived,
 
 				timing: {
-					startTime: session.timing.startTime,
-					endTime: session.timing.endTime,
+					created: session.timing.created,
+					lastRequestStarted: session.timing.lastRequestStarted,
+					lastRequestEnded: session.timing.lastRequestEnded,
 				},
 
 				statistics: session.statistics,
 			}));
 
-		this.storageService.store(AgentSessionsCache.SESSIONS_STORAGE_KEY, JSON.stringify(serialized), StorageScope.WORKSPACE, StorageTarget.MACHINE);
+		this.storageService.store(AgentSessionsCache.SESSIONS_STORAGE_KEY, JSON.stringify({
+			version: AgentSessionsCache.SESSIONS_STORAGE_VERSION,
```

Comment:
@bpasero If you prefer I can instead update the key we use to store the sessions so that we don't need a version check here

Reply:
Yes that sounds good to me, I do not think we need to migrate. Sorry for not having made that clear before. Use `agentSessions.sessions.cache` as key.